test/widgets/message_list_test.dart

```diff
@@ -1758,6 +1759,79 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
+    group('User status', () {
+      void checkFindsStatusEmoji(WidgetTester tester, Finder emojiFinder) {
+        final statusEmojiFinder = find.ancestor(of: emojiFinder,
+          matching: find.byType(UserStatusEmoji));
+        check(statusEmojiFinder).findsOne();
+        check(tester.firstWidget<UserStatusEmoji>(statusEmojiFinder)
```

Comment:
nit: since we're expecting this to find just one widget anyway, can simplify `firstWidget` to `widget`:
```suggestion
        check(statusEmojiFinder).findsOne();
        check(tester.widget<UserStatusEmoji>(statusEmojiFinder)
```